src/main/resources/com/google/api/codegen/py/versioned_directory__init__.py.snip

```diff
@@ -0,0 +1,52 @@
+@extends "py/common.snip"
+
+@snippet generate(view)
+    {@header(view.fileHeader)}
+
+    {@body(view.packageHasEnums,view.requireViews)}
+@end
+
+@private header(fileHeader)
+    {@licenseSection(fileHeader)}
+
+    {@renderImportSection(fileHeader.importSection)}
+@end
+
+@private licenseSection(fileHeader)
+    @join line : fileHeader.copyrightLines
+        @# {@line}
+    @end
+    {@BREAK}@#
+    @join line : fileHeader.licenseLines
+        @# {@line}
+    @end
+@end
+
+@private body(hasEnums, requireViews)
+    {@clients(hasEnums, requireViews)}
+
+
+    {@exports(hasEnums,requireViews)}
+@end
+
+@private clients(hasEnums, requireViews)
+    @join view : requireViews on BREAK.add(BREAK)
+        class {@view.localName}({@view.clientName}):
+                __doc__ = {@view.clientName}.__doc__
+                @if hasEnums
+                    enums = enums
+                @end
+    @end
+@end
+
+@private exports(hasEnums, requireViews)
+    __all__ = (
+        @if hasEnums
+            'enums',
+        @end
+        'types',
+        @join view : requireViews on ",".add(BREAK)
+            '{@view.localName}'
```

Comment:
I would prefer always have the trailing comma. I _assume_ that is...

```
@join view : requireViews on "".add(BREAK)
  '{@view.localName}',
@end
```

If I am wrong and it is hard, ignore this point. I am NOT asking to block on this.